### ISSUE TITLE
Not able to select MacOS Load

### ISSUE THREAD

**Issue by Nv7-GitHub** [sqweek/dialog#50]
All button are grayed out and you are not allowed to do anything. My code is following:
```go
directory, _ := dialog.Directory().Title("Load images").Browse()
```
Image:
![image](https://i.imgur.com/NArgpIN.png)

It looks like you can select the directories, but they don't do anything. The only two things on this window that work are Cancel and Open.

**Comment by Nv7-GitHub** [comment]
I found that it does start working when you put it in a MacOS app bundle. I still feel like creating an app bundle every time I change something is quite inconvenient, and I would love it if it worked without one.

**Comment by sqweek** [comment]
Is it only the directory chooser which acts this way? I'm wondering if you've stumbled on #15

**Comment by Nv7-GitHub** [comment]
No, the file loading dialog also does this. The only dialog that works is the saving dialog. But for some reason, these work when placed inside an App bundle.

**Comment by sqweek** [comment]
Yeah I dunno, it definitely works over here without an app bundle. Can you try `go run example/wde/main.go` from your `github.com/sqweek/dialog` directory? Clicking on the red box will open a load file dialog, and if I edit main.go (around line 40) to spawn a directory chooser I can interact with it no problems.

**Comment by sqweek** [comment]
Lacking further details, I'm optimistically calling this fixed by #81.
